service.go

```diff
@@ -10,12 +10,13 @@ import (
 	"github.com/micro/go-micro/cmd"
 	"github.com/micro/go-micro/metadata"
 	"github.com/micro/go-micro/server"
+	"sync"
```

Comment:
please inline this with the other go stdlib packages

Reply:
OK